src/ChronosDate.php

```diff
@@ -272,6 +270,83 @@ public static function diffFormatter(?DifferenceFormatterInterface $formatter =
         return static::$diffFormatter = $formatter;
     }
 
+    /**
+     * Add an Interval to a Date
+     *
+     * Any changes to the time will be ignored and reset to 00:00:00
+     *
+     * @param \DateInterval $interval The interval to modify this date by.
+     * @return static A modified Date instance
+     */
+    public function add(DateInterval $interval): static
+    {
+        $new = clone $this;
+        $new->native = $new->native->add($interval)->setTime(0, 0, 0);
+
+        return $new;
+    }
+
+    /**
+     * Subtract an Interval from a Date.
+     *
+     * Any changes to the time will be ignored and reset to 00:00:00
+     *
+     * @param \DateInterval $interval The interval to modify this date by.
+     * @return static A modified Date instance
+     */
+    public function sub(DateInterval $interval): static
+    {
+        $new = clone $this;
```

Comment:
Should probably assert that hours, minutes second and microseconds aren't set.

Reply:
Isn't resetting the time components enough to prevent the time from being retained? Or are you thinking of throwing an error if the interval includes hours/minutes/seconds/microseconds?

Reply:
I was thinking an assert in the interval has those set. The idea is to make users aware of input that isn't kept. However, if you think users will use intervals created from Chronos and not ChronosDate objects, then we can ignore it.

Reply:
:+1: Good idea, I've made that change and also updated `modify()` so that it also disallows modification of time components for consistency.